editor/src/components/canvas/ui-jsx-canvas-renderer/ui-jsx-canvas-component-renderer.tsx

```diff
@@ -341,20 +341,35 @@ export function createComponentRendererComponent(params: {
   return Component
 }
 
+function isRenderProp(prop: any): prop is { [UTOPIA_PATH_KEY]: string; props: MapLike<any> } {
+  return (
+    prop != null &&
+    typeof prop === 'object' &&
+    prop.props != null &&
+    typeof prop.props === 'object' &&
+    typeof prop.props[UTOPIA_PATH_KEY] === 'string'
+  )
+}
+
 // Checks if the element with the given elementPath is rendered in the props.children subtree
 // LIMITATION: this function only checks props.children, so if the given element is rendered, but from a
 // different prop, isElementInChildrenPropTree will return false
```

Comment:
I removed it whole sale in https://github.com/concrete-utopia/utopia/pull/5559/commits/a4490d7e0188b4e9f01098f9aff0100ca9db995b because now the limitation is gone